Makefile

```diff
@@ -1,35 +1,45 @@
-PYTHON_VERSIONS:=3.6.15,3.7.15,3.8.14,3.9.14,3.10.8
+PYTHON_VERSIONS:=3.13
 
 export PYTHON_VERSIONS
 
-.PHONY: test docs
+.PHONY: test docs bash rebuild
+
+build: Dockerfile docker-compose.yml
+	@docker-compose build app
```

Comment:
mixed use of `docker-compose` (v1) and `docker compose` (v2) throughout- i'd recommend sticking with the latter so it guarantees v2 regardless of user settings

Reply:
Oh right good catch.